### ISSUE TITLE
No stripped wood pillar and pillar base recipes from Totemic

### ISSUE THREAD

**Issue by ElPandaP** [Divine-Journey-2/Divine-Journey-2#899]
### Modpack version

2.18.0

### Description

There is no crafting recipe for stripped pillars and pillars bases from totemic, bark stripper doesn't work neither (it only works with normal red cedar wood blocks)

I'm not sure if this counts as a bug or as a missing feature, if it's so sorry for the wrong label :P 

### Steps to reproduce

.

### Expected behavior

The bark stripper should be able to strip the pillars or at least they should have a craft

### Your Divine Journey 2 Discord Username

ElPandaP#5326

**Comment by KleinGrrmpf** [comment]
You strip the logs with the totem knife

**Comment by ElPandaP** [comment]
> You strip the logs with the totem knife

No, with the totem knife u do totems, i mean pillar (decorative item). They can be made with normal wood but not the stripped version
![image](https://user-images.githubusercontent.com/116599304/214442597-b2ee1bfb-e5a2-4c5c-8f28-4ef4f0bd6f89.png)

**Comment by WaitingIdly** [comment]
The stripper can strip wood from only full Red Cedar Wood Logs, and its done in-world by holding right-click.
None of the other stuff is obtainable, weirdly. So, I've hidden the other stuff and added recipes for the pillars & JEI integration for the Bark Stripper
